alf/utils/common.py

```diff
@@ -1481,10 +1478,13 @@ def get_alf_snapshot_env_vars(root_dir):
     points to the ALF snapshot under this directory.
     """
     unzip_alf_snapshot(root_dir)
-    alf_repo = os.path.join(root_dir, "alf")
+    # legacy alf repo path for backward compatibility
+    legacy_alf_repo = os.path.join(root_dir, "alf")
+    alf_repo = root_dir
```

Comment:
this will make alf_examples not working for legacy alf repo

Reply:
> this will make alf_examples not working for legacy alf repo

Right, I've fixed this